### ISSUE TITLE
Docker container crash - 

### ISSUE THREAD

**Issue by tnielsen2** [ytti/oxidized#1319]
Hello,

I have been trying to upgrade oxidized to the latest version by using the most current container. However, whenever I run the container I get the following crash:

```
oxidized[1]: Oxidized starting, running as pid 1
oxidized[1]: Hook "push_to_remote" registered GithubRepo for event :post_store
oxidized[1]: lib/oxidized/nodes.rb: Loading nodes
oxidized[1]: Oxidized crashed, crashfile written in /root/.config/oxidized/crash
wrong number of arguments (given 1, expected 0)
/var/lib/gems/2.3.0/gems/oxidized-0.21.0/lib/oxidized/source/csv.rb:21:in `load': wrong number of arguments (given 1, expected 0) (ArgumentError)
	from /var/lib/gems/2.3.0/gems/oxidized-0.21.0/lib/oxidized/nodes.rb:15:in `block in load'
	from /var/lib/gems/2.3.0/gems/oxidized-0.21.0/lib/oxidized/nodes.rb:128:in `synchronize'
	from /var/lib/gems/2.3.0/gems/oxidized-0.21.0/lib/oxidized/nodes.rb:128:in `with_lock'
	from /var/lib/gems/2.3.0/gems/oxidized-0.21.0/lib/oxidized/nodes.rb:10:in `load'
	from /var/lib/gems/2.3.0/gems/oxidized-0.21.0/lib/oxidized/nodes.rb:123:in `initialize'
	from /var/lib/gems/2.3.0/gems/oxidized-0.21.0/lib/oxidized/core.rb:14:in `new'
	from /var/lib/gems/2.3.0/gems/oxidized-0.21.0/lib/oxidized/core.rb:14:in `initialize'
	from /var/lib/gems/2.3.0/gems/oxidized-0.21.0/lib/oxidized/core.rb:4:in `new'
	from /var/lib/gems/2.3.0/gems/oxidized-0.21.0/lib/oxidized/core.rb:4:in `new'
	from /var/lib/gems/2.3.0/gems/oxidized-0.21.0/lib/oxidized/cli.rb:12:in `run'
	from /var/lib/gems/2.3.0/gems/oxidized-0.21.0/bin/oxidized:8:in `<top (required)>'
	from /usr/local/bin/oxidized:23:in `load'
	from /usr/local/bin/oxidized:23:in `<main>'
[tnielsen@corp.yamww.com@yitoxidize docker-oxidized]$
```

My config and router.db files remain the same between versions.  I have made a minor modification to the Dockerfile, but only to add a directory for my local git repo. The full Dockerfile is below:

```
# -- stage 1: backport libssh2 1.7.0 from zesty for xenial, as githubrepo hook requires it
FROM ubuntu:xenial as libssh2-backport

# set up dependencies for the build process
RUN apt-get -yq update && \
    apt-get -yq install build-essential chrpath debhelper dh-autoreconf libgcrypt20-dev zlib1g-dev

# build libssh2 1.7.0
WORKDIR /tmp/libssh2-build
ADD https://launchpad.net/ubuntu/+archive/primary/+files/libssh2_1.7.0-1ubuntu1.debian.tar.xz .
ADD https://launchpad.net/ubuntu/+archive/primary/+files/libssh2_1.7.0.orig.tar.gz .
RUN tar xvf libssh2_1.7.0.orig.tar.gz
WORKDIR /tmp/libssh2-build/libssh2-1.7.0
RUN tar xvf ../libssh2_1.7.0-1ubuntu1.debian.tar.xz

WORKDIR /tmp/libssh2-build/libssh2-1.7.0
ENV DEB_BUILD_OPTIONS nocheck
RUN dpkg-buildpackage -b

# -- stage 2: build the actual oxidized container
FROM phusion/baseimage:0.10.0
LABEL maintainer="Samer Abdel-Hafez <sam@arahant.net>"

# set up dependencies for the build process
RUN apt-get -yq update && \
    apt-get -yq install ruby2.3 ruby2.3-dev libsqlite3-dev libssl-dev pkg-config make cmake libssh2-1-dev git g++ libffi-dev ruby-bundler

# upgrade libssh2 to self-built backport from stage 1
COPY --from=libssh2-backport \
    /tmp/libssh2-build/libssh2-1_1.7.0-1ubuntu1_amd64.deb \
    /tmp/libssh2-build/libssh2-1-dev_1.7.0-1ubuntu1_amd64.deb \
    /tmp/
RUN dpkg -i /tmp/*.deb

# dependencies for hooks
RUN gem install aws-sdk slack-api xmpp4r cisco_spark

# build and install oxidized
COPY . /tmp/oxidized/
WORKDIR /tmp/oxidized

# docker automated build gets shallow copy, but non-shallow copy cannot be unshallowed
RUN git fetch --unshallow || true
RUN rake install

# web interface
RUN gem install oxidized-web --no-ri --no-rdoc

# clean up
WORKDIR /
RUN rm -rf /tmp/oxidized
RUN rm /tmp/*.deb
RUN apt-get -yq --purge autoremove ruby-dev pkg-config make cmake ruby-bundler

# add runit services
ADD extra/oxidized.runit /etc/service/oxidized/run
ADD extra/auto-reload-config.runit /etc/service/auto-reload-config/run
ADD extra/update-ca-certificates.runit /etc/service/update-ca-certificates/run

# add directories for git repo
RUN mkdir -p /root/.config/oxidized/oxidized-configs

VOLUME ["/root/.config/oxidized"]
EXPOSE 8888/tcp
```

Lastly, here is my config file:

```
---
username: OMITTED
password: OMITTED
model: cisco
interval: 1800
use_syslog: true
log: /root/.config/oxidized/logs
debug: true
threads: 50
timeout: 960
retries: 1
prompt: !ruby/regexp /^([\w.@-]+[#>]\s?)$/
rest: 0.0.0.0:8888
next_adds_job: false
vars:
  remove_secret: false
  enable: OMITTED
groups: {}
models: {}
pid: /root/.config/oxidized/pid
input:
  default: ssh, telnet
  debug: true
  ssh:
    secure: false

output:
  default: git
  git:
    single_repo: true
    user: root
    email: OMITTED
    repo: /root/.config/oxidized-configs/.git

source:
  default: csv
  csv:
    file: /root/.config/oxidized/router.db
    delimiter: !ruby/regexp /:/
    map:
      name: 0
      ip: 1
      model: 2
      username: 3
      password: 4
      enable: 5
    gpg: false

model_map:
  cisco: ios
  cisco: asa
  juniper: junos
  ciscowlc: aireos
  brocade: nos
  fortigate: fortios
  sonicwall: aosw
  ironware: ironware

hooks:
  push_to_remote:
   type: githubrepo
   events: [post_store]
   remote_repo: OMITTED
   username: tnielsen2
   password: OMITTED

```

**Comment by wk** [comment]
Proposed fix in #1320, please test if you're able.

**Comment by tnielsen2** [comment]
This fixed the issue. Thank you very much.
